### ISSUE TITLE
lint: add if/else -> switch suggesting checker

### ISSUE THREAD

**Issue by Quasilyte** [PieselBois/kfulint#77]
If there is a chain of if/else, suggest switch as alternative.

These two functions should trigger the warning:
```go

func cond1(x int) string {
	if x == 0 {
		return "zero"
	} else if x < 0 {
		return "negative"
	} else {
		return "positive"
	}
}

func cond2(x int) string {
	if x == 0 {
		return "zero"
	} else if x < 0 {
		return "negative"
	}
	return "positive"
}
```
Suggested alternative is:
```go
func condSuggested(x int) string {
	switch {
	case x == 0:
		return "zero"
	case x < 0:
		return "negative"
	default:
		return "positive"
	}
}
```

TODO: find out if `if/else` is ever more readable or wanted instead of switch. 
Maybe init statements can complicate things.
